polygeist/tools/cgeist/Lib/clang-mlir.h

```diff
@@ -348,6 +348,11 @@ class MLIRScanner : public clang::StmtVisitor<MLIRScanner, ValueCategory> {
                      llvm::Optional<mlir::Type> ReturnType,
                      llvm::StringRef MangledFunctionName);
 
+  /// Creates an instance of a SYCL grid operation replacing a call to \param
```

Comment:
Do we need a declaration in the header file ? I see that `createSYCLBuiltinOp` is only used in `CGExpr.cc`.

Reply:
Is OK.